tofhir-server/src/main/scala/io/tofhir/server/fhir/FhirDefinitionsConfig.scala

```diff
@@ -7,6 +7,8 @@ import scala.util.Try
 
 class FhirDefinitionsConfig(fhirDefinitionsConfig: Config) {
 
+  /** Major FHIR version (R4 or R5) */
+  lazy val majorFhirVersion: String = Try(fhirDefinitionsConfig.getString("fhir-version")).getOrElse("R5")
```

Comment:
If we are planning to use R5 by default, let's change other files accordingly including the fhir-version settings in application.conf file. If the default version is R4, let's change it **getOrElse("R4")**